task/hiffy/src/main.rs

```diff
@@ -61,13 +61,16 @@ cfg_if::cfg_if! {
         const HIFFY_RSTACK_SIZE: usize = 2048;
     }
 }
+const HIFFY_SCRATCH_SIZE: usize = 256;
```

Comment:
As a result of this change you've made it so that `HIFFY_SCRATCH_SIZE` could usefully be changed -- since Humility can detect its size now. I'd suggest adding a comment on this constant to that effect, something like

```rust
/// Number of "scratch" bytes available to Hiffy programs. Humility uses this to deliver data
/// used by some operations. This number can be increased at the cost of RAM.

Reply:
Done.